src/components/Tree/Tree.tsx

```diff
@@ -16,6 +16,8 @@ export interface TreeFlatListItem {
     value?: string;
     actions?: React.ReactNode[];
     parentId: NullableString;
+    editable?: boolean;
+    onEditableSave?: (value: string) => void;
 }
```

Comment:
We were discussing about the properties. Do you think it would be enough to just have the Callback property instead of both?

Reply:
It would be, I am also not a fan of having both properties. However, consistency wise, I think editable is a good prop to have.

Guess it's your call in the end 😄

Reply:
Then we leave for now if you see that it helps consistency